### ISSUE TITLE
Request for Office Addin Sample for Blazor with SSO integration

### ISSUE THREAD

**Issue by bkotar1** [OfficeDev/Office-Add-in-samples#672]
I am unable to find a proper sample for SSO for blazor Office Addin and it will be helpful if someone could create a sample with SSO integration with Azure AD.

**Comment by aafvstam** [comment]
That is true as the Blazor Office Add-in is fairly new (I came up with the concept and the Blazor Samples in here). I can try to spend some time to see if I can get that to work. I have no clue if it is supported but can ask around if it is feasible.

**Comment by davidchesnut** [comment]
Hi @bkotar1, thank you for this suggestion. I tagged this as a doc request (sampler request) for now. SSO should work with Blazor, but as @aafvstam points out we don't have a sample showing both at this time. 

Thanks!
David

**Comment by bkotar1** [comment]
Thanks @aafvstam  and @davidchesnut.
@davidchesnut Yes, I see we can use the outlook addin sso sample and integrate into blazor using msal js libraries. Just want to see if we have any other options in integrating it. As always samples will help in quickly starting the project without worrying much in integrating the msal js libraries.

**Comment by PaulYork-GitHub** [comment]
Hi, 

I'd also love to see a working example of this. I've been trying various examples but feel like I'm missing something so anything that proceeds a clear working example would be great.
Thanks,
Paul.

**Comment by ystvan** [comment]
I second this, seeing rudimentary sample of SSO & Graph Client in a .NET 8 Blazor WASM and hosted WASM would be great and much appreciated!
